src/RegistrationController.js

```diff
@@ -128,15 +128,19 @@ function (
       });
       this.model = new Model();
 
-      var checkPasswordMeetComplexities = function(model) {
+      var checkPasswordMeetComplexities = function (model, showError) {
         var password = model.get('password') || '';
         _.each(schema.passwordComplexity.enabledComplexities, function(complexityName) {
-          var ele = Okta.$('.password-complexity-' + complexityName);
+          var ele = Okta.$('#password-complexity-' + complexityName);
           var complexityValue = this.get(complexityName);
+          ele.removeClass();
```

Comment:
Does this element have classes other than the ``password-complexity-*``? This will remove all classes on this element, including other ones if they're present.